Formula/bullet.rb

```diff
@@ -26,7 +25,7 @@ class Bullet < Formula
   def install
     args = std_cmake_args + %w[
       -DINSTALL_EXTRA_LIBS=ON -DBUILD_UNIT_TESTS=OFF -DBUILD_PYBULLET=OFF
-    ]
+    ] + ["-DCMAKE_INSTALL_RPATH=#{lib}"]
```

Comment:
Why not make it part of the list above?

Reply:
Hi, thanks for the review. I tried that first but the %w inhibits the evaluation of the lib variable.

Reply:
@fredowski Does it work if you use `%W`?